main/main.go

```diff
@@ -21,7 +21,7 @@ import (
 var (
 	versionMajor    = "1"
 	versionMinor    = "1"
-	versionBuild    = "0"
+	versionBuild    = "1"
```

Comment:
Version update should be a separate PR

Reply:
Last time, I have received comment to change it in same PR. So, I included this time.

Reply:
Sorry about that, I was wrong that time :)

Reply:
No worries, I will undo and open separate PR.